target_score.txt

```diff
@@ -5,4 +5,4 @@
 6
```

Comment:
pipeline should not be checked if the mark is 0 at line 5 (that is score for pipeline) and not if the commit message has the proper mark!

Reply:
fixed